okhttp/src/main/kotlin/okhttp3/Authenticator.kt

```diff
@@ -114,5 +115,9 @@ interface Authenticator {
     private class AuthenticatorNone : Authenticator {
       override fun authenticate(route: Route?, response: Response): Request? = null
     }
+
+    /** An authenticator that knows no credentials and makes no attempt to authenticate. */
```

Comment:
Copy pasta problem

This has credentials; they’re installed in `java.net.Authenticator` !